assets/src/edit-story/components/inspector/prepublish/components/checklistTab.js

```diff
@@ -243,81 +212,81 @@ const ChecklistTab = ({
     [renderRow]
   );
 
+  // Determine what to make visible based on checkpoint and present issues.
   const highPriorityLength =
     highPriority.length + (pages.lengths?.highPriority || 0);
+  const hasHighPriorityItems = Boolean(highPriorityLength);
+  const isHighPriorityVisible =
+    !isHighPriorityDisabledState && hasHighPriorityItems;
+
   const recommendedLength =
     recommended.length + (pages.lengths?.recommended || 0);
-  const hasHighPriorityItems = Boolean(highPriorityLength);
   const hasRecommendedItems =
-    Boolean(recommended.length) || Boolean(pages.lengths?.recommended);
+    Boolean(recommended.length) ||
+    Boolean(pages.lengths?.recommended) ||
+    (canOptimizeVideo && !areVideosAutoOptimized);
```

Comment:
This is new here, even if recommended items are visible now and there are none to show if the optimize video experiment is true and an option for the user we still need to show the recommended section because of current structure of the checklist. So this is adding a secondary check for that condition.